Python/Product/PythonTools/PythonTools/Repl/InteractiveWindowProvider.cs

```diff
@@ -91,13 +94,27 @@ class InteractiveWindowProvider {
             return false;
         }
 
-        public IVsInteractiveWindow Open(string replId) {
+        public void OnWindowUsed(IVsInteractiveWindow window) {
+            if (window == null) {
+                throw new ArgumentNullException(nameof(window));
+            }
+
+            lock (_windows) {
+                _lruWindows.Remove(window);
+                _lruWindows.Add(window);
```

Comment:
What does `lru` mean?

Reply:
Least-recently used (opposite order of MRU)

Reply:
(To make it MRU we would `Insert(0, window)` instead of `Add(window)`, which takes longer)